src/Firehose.Web/Authors/MalcolmJack.cs

```diff
@@ -22,5 +25,16 @@ public IEnumerable<Uri> FeedUris
         public string StateOrRegion => "Cape Town, South Africa";
         public string TwitterHandle => "inquisitorjax";
         public Uri WebSite => new Uri("https://www.inquisitorjax.blogspot.com/");
```

Comment:
Do we enforce the HTTPs requirement on the websites as well? Because this cert doesnt look valid..

Reply:
the non-www version is ok: https://inquisitorjax.blogspot.dk